packages/localize/src/number/parseNumber.js

```diff
@@ -1,56 +1,27 @@
 import { getDecimalSeparator } from './getDecimalSeparator.js';
 
-/**
- * @param {string} value to evaluate
- * @return {boolean} true if value equal . or ,
- */
-function isDecimalSeparator(value) {
-  return value === '.' || value === ',';
-}
-
 /**
  * Determines the best possible parsing mode.
- *
- * - If there is only one separator (withLocale)
- * - 1,23 => xxx1.23 (heuristic)
- * - else parse mode depends mostly on the last 4 chars
- * - 1234 => xxx1234 (heuristic)
- * - [space]123 => xxx123 (heuristic)
- * - ,123 => unclear
- *   - if 1.000,123 (we find a different separator) => 1000.123 (heuristic)
- *   - if 1,000,123 (we find only same separators) => 1000123 (unparseable)
- *   - if 100,123 (we find no more separators) => unclear
- *     - if en => 100123 (withLocale)
- *     - if nl => 100.123 (withLocale)
- *
```

Comment:
Would it be possible to update this to the new outcomes? It's quite important for understanding all this

Reply:
Also this needs to be updated? https://github.com/ing-bank/lion/blob/42f7026b5883e276859060008c4af15be79e20d8/packages/localize/src/number/parseNumber.js#L114

Reply:
This list was very unreadable, while the description of the parseNumber was much easier to understand.
Both descriptions were not inline with the old code, but are inline with the updated code.